doc/conceptualOverview.dox

```diff
@@ -57,30 +56,161 @@ Poll inputs do not have the AccessMode::wait_for_new_data flag, and hence all re
 
-\section conceptualOverview_Application The Application
+This will publish all registers from the catalogue of the specified device "oven" to the control system and to the application modules. Since "oven" is a device alias and not a CDD, we also need to specify the DMAP file by creating an instance of SetDMapFilePath *before* the DeviceModule instance like this:
+
+\snippet{lineno} example/include/ExampleApp.h SetDMapFilePath
+
+The call to getName() returns the name of the Application, which will be provided later to the Application constructor. This allows us to influence the file name for automated tests. The DMAP file in our exmaple speficies two devies:
 
-\section conceptualOverview_Connections Process variable connections
+\include example/config/demo_example.dmap
+
+It is a good practise to use a <a href="https://chimeratk.github.io/DeviceAccess/master/lmap.html">logical name mapping device</a> to allow more control over the representation of the device. The second device is the actaul hardware device which will be used as a target for the logical name mapping device. We are using a sharedMemoryDummy backend (map file \ref example_config_demo_example_dmap "DemoDummy.map"), so we can run the application locally and interact with the dummy device through QtHardMon.
+
+The \ref example_config_oven_xlmap "example xlmap mapping file" generates the following variable hierarchy:
+
+- /Configuration/heaterMode (redirects to to HEATER.MODE, read/write)
+- /Configuration/lightOn (redirects to BOARD.GPIO_OUT0, read/write)
+- /Controller/heatingCurrent (reditects to HEATER.CURRENT_SET, read/write)
+- /Controller/temperatureReadback (redirects to SENSORS.TEMPERATURE1, read only)
+- /Monitoring/heatingCurrent (redirects to HEATER.CURRENT_READBACK, read only)
+- /Monitoring/temperatureOvenTop (redirects to SENSORS.TEMPERATURE2, read only)
+- /Monitoring/temperatureOvenBottom (redirects to SENSORS.TEMPERATURE3, read only)
+- /Monitoring/temperatureOutside (redirects to SENSORS.TEMPERATURE4, read only)
+
+Since in ApplicationCore, all device registers are treated as unidirectional, read/write registers will be used in the write direction only. Hence, the first 3 variables will have the device as a consumer, receiving values either from an ApplicationModule or from the control system. The other variables will use the device as a feeder, so the device will provide values to the ApplicationModules and the control system.
+
+In case of our example, the device expects the application to poll the data (no registers support AccessMode::wait_for_new_data as the device does not support interrupts). Hence, the name of a push-type variable triggering the readout needs to be specified as a 3rd argument to the DeviceModule. This trigger variable will affect only registers which need a trigger and can even come from the device itself, in case it provides a data ready interrupt which shall be used to trigger the readout.
```

Comment:
At this point you are already half a screen away from code line 49, where you are describing the DeviceModule initialisation, which you are referring to here. You made a detour to logical name mapping, which is not C++ and already part of DeviceAccess (although essential for the ApplicationCore working principle).
Maybe put a lead in sentence? "After the detour to xlmap we are back to C++ code line 49...."

Reply:
done